bin/create-gke-cluster.sh

```diff
@@ -8,10 +8,11 @@ readonly CLUSTER
 readonly METERING_SET=${METERING_SET:-otomi_metering}
 readonly VERBOSE=${VERBOSE:-0}
 
-readonly PROJECT_ID=${PROJECT_ID:-$(yq read $ENV_DIR/env/clusters.yaml google.projectId)}
+readonly PROJECT_ID=${PROJECT_ID:-$(yq read $ENV_DIR/env/clusters.yaml clouds.google.projectId)}
```

Comment:
this is NOT correct now, as it is now part of the root `google` prop still (even though that might not be smart, as it should be possible to use multiple projectIds for clusters.

Reply:
the problem with these values was that they were missing and I had to add them.

Reply:
And this way of getting values `$(yq read $ENV_DIR/env/clusters.yaml)` is confusing and very misleading as most of the `secret` values need to be rendered from helm templating

Reply:
`google.projectId` and `clouds.google.clusters...` should be separate blocks in `clusters.yaml` then